src/js/components/form/FieldAutofocus.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import DOMUtils from "#SRC/js/utils/DOMUtils";
+
+let lockFieldAutofocus = false;
```

Comment:
I understand. That makes sense, and I was able to easily reproduce that use case locally. Thanks for clearing it up!